src/components/sidebar/Authors.jsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
-import Author from './Author'
+import AuthorThumb from './AuthorThumb'
 
 const Authors = ({ authorImages, authors, setAuthor }) => {
-  const sortedAuthors = [...authors].sort()
-  const preparedAuthors = sortedAuthors.map((author, i) =>
-    <Author key={i} authorName={author} authorImage={authorImages.get(author)} setAuthor={setAuthor} />
+  const preparedAuthors = [...authors].sort().map((author, i) =>
+    <AuthorThumb key={i} authorName={author} authorImage={authorImages.get(author)} setAuthor={setAuthor} />
```

Comment:
ne bi trebalo index da se koristi kao key. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Znam, ali nemamo drugo uniq polje, jel imaš ideju šta bi trebalo?

Reply:
ustvari, videh u clanku dodao sam shortid

Reply:
izgleda da nesto nisam uradio kako treba, sad mi izbacuje `Warning: Can only update a mounted or mounting component. This usually means you called setState, replaceState, or forceUpdate on an unmounted component. This is a no-op.`